code/game/objects/machinery/roomba.dm

```diff
@@ -49,6 +49,17 @@
 		return
 	step_to(src, get_step(src,newdir))
 
+/obj/machinery/roomba/attack_hand(mob/living/user)
+	if(!CONFIG_GET(flag/fun_allowed))
+		return
+	if(user.a_intent != INTENT_HARM)
+		return
+	tgui_alert(user, "Are you really sure to want to try your luck with the devilish roomba?", "The roomba roulette", list("Yes", "Yes!", "Yes?"))
+	if(prob(50))
+		explosion(user, 1, 0, 0, 0, 0, 4, "[user] lost at the roomba roulette")
+		return
+	explosion(src, 1, 0, 0, 0, 0, 4, "[user] won at the roomba roulette")
+	qdel(src)
```

Comment:
Imagine not turning the person into bread that the roomba will _**C O N S U M E**_

Reply:
yeah ok

Reply:
Dis good enough for a shit post